mappings/net/minecraft/world/level/LevelPropertiesImpl.mapping

```diff
@@ -0,0 +1,50 @@
+CLASS net/minecraft/class_31 net/minecraft/world/level/LevelPropertiesImpl
```

Comment:
`LevelProperties`, no need to rename this freaking class as this is still the exact representation of `level.dat` file